components/ui/toggle.tsx

```
@@ -35,4 +35,4 @@ const Toggle = React.forwardRef<
 
 Toggle.displayName = TogglePrimitive.Root.displayName;
 
-export { Toggle, toggleVariants };
+export { Toggle,  };
```

ESLint

[error] Delete `,·`

(prettier/prettier)